code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -548,52 +548,88 @@
 	reagent_state = LIQUID
 	color = "#D2FFFA"
 	metabolization_rate = 0.5 * REAGENTS_METABOLISM
-	overdose_threshold = 45
-	addiction_threshold = 30
+	overdose_threshold = 30
+	addiction_threshold = 25
 
 /datum/reagent/medicine/ephedrine/on_mob_add(mob/living/L)
 	..()
-	L.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown=-1, blacklisted_movetypes=(FLYING|FLOATING))
+	L.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown=-0.85, blacklisted_movetypes=(FLYING|FLOATING))
 
 /datum/reagent/medicine/ephedrine/on_mob_delete(mob/living/L)
 	L.remove_movespeed_modifier(id)
 	..()
 
 /datum/reagent/medicine/ephedrine/on_mob_life(mob/living/carbon/M)
+	if(prob(20) && iscarbon(M))
+		var/obj/item/I = M.get_active_held_item()
+		if(I)
```

Comment:
```suggestion
		if(I && M.dropItemToGround(I))
```

Reply:
the to_chat should only work if you actually drop the item (else you might get instances where you have a nodrop but you still get the spaghetti message)